symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -610,13 +631,15 @@ protected function getSectionAttachments()
                      . $this->contentInfoString($file['mime-type'], $file['file'], $file['filename'], $file['charset'])
                      . EmailHelper::base64ContentTransferEncode($file_content);
             } else {
-                if (!$tmp_file === false) {
-                    $filename = $original_filename;
-                } else {
-                    $filename = $file['file'];
+                if ($this->_ignore_attachment_errors === false) {
+                    if (!$tmp_file === false) {
```

Comment:
You mean changing `if (!$tmp_file === false)`? It has been like that before. I kept the existing code an just wrapped it into `if ($this->_ignore_attachment_errors === false)`. I think that changing the old code might be part of a cleanup, but I wouldn't like to do this in a PR like this one. The PR should only have one topic.